packages/nimble-components/src/table-column/enum-base/tests/shared.ts

```diff
@@ -0,0 +1,9 @@
+export const sharedMappingValidityDescription = `Readonly object of boolean values that represents the validity states that the column's configuration can be in.
```

Comment:
Can you name this file something more descriptive, like `shared-storybook-docs.ts`